app/assets/javascripts/admin/addon/components/site-settings/file-size-restriction.js

```diff
@@ -0,0 +1,15 @@
+import Component from "@ember/component";
+import { action } from "@ember/object";
+
+export default class FileSizeRestriction extends Component {
+  @action
+  onChangeSize(size) {
+    //this.set("validationMessage", "asdfasdf");
```

Comment:
```suggestion
```

but you can use `{{fn (mut @value)}}` and `{{fn (mut @validationMessage)}}` to make this component template-only

Reply:
Maybe I did something wrong, but I just tried this and it didn't work.

Reply:
Nevermind, got it working for value, but for validationMessage it's a bit different now.